Makefile

```diff
@@ -76,6 +76,7 @@ INSTALL_DATA = $(INSTALL) -m644
 
 # Toolchain
 MSBUILD = msbuild -verbosity:m -nologo
+DOTNET = dotnet build
```

Comment:
We should probably use
```
DOTNET = dotnet
DOTNET_BUILD = $(DOTNET) build
```

and then use `$(DOTNET)` when running `check` and `clean` commands and `$(DOTNET_BUILD)` when compiling... or maybe use `$(DOTNET)` for all and add the `build` argument in the compile lines instead of separately defining `$(DOTNET_BUILD)`?